packages/docs/content/utilities/ring.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Ring
+---
+
+import { Button } from '@cmsgov/design-system';
+
+Use the following utilities to apply focus styles to custom UI components.
+
+## Examples
+
+### Focus rings
+
+**Focus**
+
+Use the `.focus:ring` utility to conditionally apply focus styles to your component when that component has focus. Focus will be applied regardless of keyboard or mouse interaction.
+
+Click or tab into the following example to see this utility in action.
+
+<EmbeddedExample>
+  <p tabIndex="0" class="focus:ring ds-u-border--1 ds-u-padding--2">Focus state example</p>
+</EmbeddedExample>
+
+**Focus-visible**
+
+Use the `.focus-visible:ring` utility to conditionally apply focus styles to your component when that component has keyboard focus.
+
+Tab into the following example to see this utility in action.
+
+<EmbeddedExample>
+  <p tabIndex="0" class="focus-visible:ring ds-u-border--1 ds-u-padding--2">Focus-visible state example</p>
+</EmbeddedExample>
```

Comment:
I can't seem to tab into this one. Maybe it's because it's a paragraph? Thinking of another possible example with an interactive element

Reply:
Never mind. User error